data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1625 @@
+			choice
+				`	"Sounds good. Time to go."`
+					goto departure
+				`	"What about people seeing the ship?"`
+			`	Chilia grins. "Fortunately for us, the diversity of human preference lends itself to our cause. While there may be only a handful of ships available for sale, spaceships are a surprisingly diverse and unregulated industry. As such, shipyards have carried on the tradition dating back to the earliest of vehicles: If someone has the wealth to pay for it, there is a shipwright willing to build it.`
+			`	In our time observing Ancestral space, we have seen replicas in flight of every spaceship that has ever graced popular culture or a media production. While many are barely able to manage interplanetary travel, a surprising number are in use as fully functional ships plying interstellar space. Given the similarity between our ships and those of certain productions, we are not concerned with being seen - only with being scanned."`
+			label departure
+			`	A hatch opens on the Peregrine, and all the workers quickly finish their tasks and stow their equipment onboard. "Ready for departure when you are, Captain!" calls one from the hatch.`
+				accept
+	on accept
+		dialog
+			`	As the trio move clear of the landing pad Chilia signs one last message to you. "You may want to cloak as soon as you take off. Our ships will be scanning for you too: Better to practice avoiding them before you reach Ancestral space. Embers guide you, <first>!"`
+		give ship Peregrine "Shadow Flight"
+		event "remnant: cognizance peregrine"
+	on decline
+		dialog
+			`	You have declined an essential Remnant mission, and this ends the Remnant campaign. If that was not what you intended, you may wish to reload.`
+
+
+
+mission "Remnant: Cognizance 32"
+	name "Pickup refined fuel."
+	description "Deliver the refined fuel to Ssil Vida by <date>."
+	landing
+	deadline 19
+	cargo "Refined Fuel" 58
+	illegal 0
+	stealth
+	source "Clink"
+	destination "Ssil Vida"
+	to offer
+		has "Remnant: Cognizance 31: done"
+	on offer
+		conversation
+			`Your reception is as friendly as Clink's reputation suggests: No-one is there to meet you, and no-one arrives to help load the cargo. Just a forlorn datapad mounted on a post in front of the pile of crates requesting that you enter your pickup code to disarm the security sensors, and a statement that ships depart as soon as they are done loading. You enter the pickup code Chilia had sent you while you were in flight, load the <ship>, and get ready to head to Ssil Vida.`
+			`	As you are completing your checks your sensors alert you to the arrival of a small fleet that seems to be maintaining orbit over this area. It looks like you might have to take steps to avoid observers right away.`
```

Comment:
```suggestion
			`	As you are completing your checks, your sensors alert you to the arrival of a small fleet that seems to be maintaining orbit over this area. It looks like you might have to take steps to avoid observers right away.`
```